specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/apigateway.json

```diff
@@ -666,6 +666,29 @@
           "type": "string",
           "description": "ETag of the resource.",
           "readOnly": true
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The kind of gateway.",
+          "enum": [
+            "KubernetesManaged",
+            "Default"
+          ],
+          "x-ms-enum": {
+            "name": "GatewayKind",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "KubernetesManaged",
+                "description": "Kind of gateway that is capable of exposing traffic routes for workspaces using configuration managed by Kubernetes resources."
+              },
+              {
+                "value": "Default",
+                "description": "Kind of gateway that is capable of exposing APIs for workspaces using configuration managed through Azure Resource Manager (ARM)."
+              }
+            ]
```

Comment:
This is a massive nit, but let's put the default first so that it's the first item in generated enums as well (and sorted alphabetically)

Reply:
Done.